packages/x-date-pickers/src/StaticNextTimePicker/StaticNextTimePicker.tsx

```diff
@@ -1,14 +1,15 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
+import { TimeView } from '@mui/x-date-pickers/internals';
 import { StaticNextTimePickerProps } from './StaticNextTimePicker.types';
 import { useNextTimePickerDefaultizedProps } from '../NextTimePicker/shared';
 import { singleItemValueManager } from '../internals/utils/valueManagers';
 import { useStaticPicker } from '../internals/hooks/useStaticPicker';
 import { renderTimeView } from '../internals/utils/viewRenderers';
 import { validateTime } from '../internals/hooks/validation/useTimeValidation';
 
-type StaticTimePickerComponent = (<TTime>(
-  props: StaticNextTimePickerProps<TTime> & React.RefAttributes<HTMLDivElement>,
+type StaticTimePickerComponent = (<TDate>(
```

Comment:
Small mistake when I copied `StaticNextDatePicker` and changed all the `Date` to `Time` :see_no_evil: